### ISSUE TITLE
Cannot read properties of undefined (reading 'flag')

### ISSUE THREAD

**Issue by ilyakooo0** [tloncorp/landscape-apps#2625]
```
TypeError: Cannot read properties of undefined (reading 'flag')
    at O3 (https://urbit.iko.soy/apps/groups/assets/index.b18819a0.js:45:108521)
    at Kl (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884)
    at Ka (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:62361)
    at Zd (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:61688)
    at mf (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:11937)
    at pf (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:947)
    at yg (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:875)
    at rn (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:730)
    at sl (https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:21:10884)
    at https://urbit.iko.soy/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38716
```

**Comment by jamesacklin** [comment]
Thanks for the report. What page were you on when this occurred? What were you doing?

**Comment by jamesacklin** [comment]
Tracking internally as LAND-594

Fixed by #2635
